src/containers/Main/index.jsx

```diff
@@ -3,15 +3,28 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 import MainComponent from '../../components/Main';
-import { fetchData } from './actions';
-
-function Main({ appData, fetchData }) {
-  return (
-    <MainComponent
-      appData={appData}
-      fetchData={fetchData}
-    />
-  );
+import { fetchData, rehydrate } from './actions';
+
+class Main extends React.Component {
+  componentWillMount() {
+    const localStorageData = JSON.parse(localStorage.getItem('appData'));
```

Comment:
Is not possible to load data from store/initialState?

Reply:
I'm not sure what do you mean...
The initialState is loaded if there's no localStorage data as you can see in the REHYDRATE reducer.

Reply:
I was thinking what happen if you set the initialState of the store with localStorage data, instead of loading the data on mount, but I wanted to know your opinion, what you're doing now is not wrong ;)

Reply:
Oh, do you mean having the localStorage getter right in the reducer or something like that?
The way I thought this is that redux is the ´source of truth´ and localStorage acts only like a cache.
So in order to apply the cache to redux's store I would need to dispatch an action.
Maybe we can discuss this in person later :)